Projects/MakeMeLaugh_Server/Assets/Scenes/MainScene.unity

```diff
@@ -1090,6 +1090,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   JoinCode: 
+  ClientHostURI: https://richard-fine.itch.io/the-laugh-hole?secret=jal3L5588KAmK3SErcfq9IpPAw
```

Comment:
do we want to check that in, I keep stashing it away :) ?

Reply:
Oh right. I think it’s fine, we should probably replace it with an environment variable at some point but for now I don’t mind checking it in.